### PR TITLE
[AMS-2020] Update sponsor page

### PR DESCRIPTION
**Pull request by FloorD**
Please check that the sponsor benefits are accurate, and the cap on attendees is correct, @PaulusTM

### CODE REVIEW

File: content/events/2020-amsterdam/sponsor.md [hunk 1]
```diff
@@ -5,13 +5,27 @@ Description = "Sponsor devopsdays Amsterdam 2020"
...
+<p>
+  
+| | Bronze | Silver | Gold 
+| --- | --- | --- | ---
+| | **1k** | **3k** | **6k**
+| Logo on website | x | x | x
+| Logo in emails | x | x | x |
+| Logo on shared break slide | x | | 
+| Logo on dedicated break slide | | x | x
+| Sponsor pitch | | | x
```
Comment (PaulusTM) [inline content/events/2020-amsterdam/sponsor.md:26]: On the image I have we call this: "Message on dedicated break slide".
"Sponsor pitch" might be better.

File: content/events/2020-amsterdam/sponsor.md [hunk 2]
```diff
@@ -5,13 +5,27 @@ Description = "Sponsor devopsdays Amsterdam 2020"
...
+devopsdays is a conference organized by volunteers, and depends on sponsorships. We do not have vendor booths, sell keynote slots, nor do we distribute attendee contact lists. However, sponsors have the opportunity to deliver an elevator pitch during the main program, and will get recognition on the website and on social media before, during and after the event. Sponsors are encouraged to represent themselves by actively participating and engaging with the attendees in the chat. Like any other attendee, sponsors have the opportunity to demo products/projects as part of an Open Space session.
 <p>	
-The best thing to do is send engineers to interact with the experts at devopsdays on their own terms.
-<p>
\ No newline at end of file
+  <strong>Hot tip:</strong> encourage your engineers to participate in, and interact with the attendees and experts at devopsdays. The in-person devopsdays Amsterdam conference typically attracts around 500 participants. For our virtual 2020 event we've capped attendance at 1000. 
+<p>
+  
+| | Bronze | Silver | Gold 
+| --- | --- | --- | ---
```
Comment (PaulusTM) [inline content/events/2020-amsterdam/sponsor.md:20]: Did you leave "Sticker in swag package" out on purpose?

**Comment by mattstratton** [comment]
@FloorD we try to avoid keeping WIP pull requests open on this repo; can you review the comments from @PaulusTM and resolve them? Otherwise, we will need to close this PR and have you resubmit it when it can be worked on. Thanks!

File: content/events/2020-amsterdam/sponsor.md (hunk 2 above)
Comment (FloorD) [reply content/events/2020-amsterdam/sponsor.md:20]: I thought we decided against this in our last meeting, but could be mistaken. @sheela-nistala ?

**Comment by PaulusTM** [comment]
Checked everything and it is ready to merge @mattstratton 
Thnx for the work @FloorD! :)

File: content/events/2020-amsterdam/sponsor.md (hunk 2 above)
Comment (PaulusTM) [reply content/events/2020-amsterdam/sponsor.md:20]: You're right.

**Pull request merged**